apps/dashboard/src/components/Plans/List.tsx

```diff
@@ -40,61 +40,57 @@ const List: React.FC<{
     [intl, toast]
   );
   const deletePlan = useDeletePlan({ onSuccess, onError });
-  const columnHelper = createColumnHelper<IPlan.MappedAttributes>();
+  const columnHelper = createColumnHelper<IPlan.Self>();
   const columns = useMemo(
     () => [
       columnHelper.accessor("id", {
         header: intl("global.labels.id"),
         cell: (info) => info.getValue(),
       }),
-      columnHelper.accessor("alias", {
-        header: intl("dashboard.plan.title"),
-        cell: (info) => info.getValue(),
-      }),
       columnHelper.accessor("weeklyMinutes", {
         header: intl("dashboard.plan.weeklyMinutes"),
         cell: (info) => formatMinutes(info.getValue()),
       }),
-      columnHelper.accessor("fullMonthPrice", {
+      columnHelper.accessor("baseMonthlyPrice", {
         header: intl("dashboard.plan.fullMonthPrice"),
         cell: (info) => {
           return (
             <Price
               price={info.getValue()}
-              discount={info.row.original.fullMonthDiscount}
+              discount={info.row.original.baseMonthlyPrice}
             />
           );
         },
       }),
-      columnHelper.accessor("fullQuarterPrice", {
+      columnHelper.accessor("monthDiscount", {
         header: intl("dashboard.plan.fullQuarterPrice"),
         cell: (info) => {
           return (
             <Price
               price={info.getValue()}
```

Comment:
The price should accept the original price before discount. 


```suggestion
              price={info.row.original.baseMontlyPrice}
```

> [!NOTE]
> Update the other columns.

Reply:
I thought you said we will remove this file!... won't we?